client/app/bundles/momentCard/components/MomentCard.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React from 'react';
+import MomentCardName from './MomentCardName';
+import MomentCardDate from './MomentCardDate';
+import MomentCardDraft from './MomentCardDraft';
+import MomentCardSettings from './MomentCardSettings';
+import MomentCardCategories from './MomentCardCategories';
+import MomentCardMoods from './MomentCardMoods';
+import Tag from '../../shared/components/Tag';
+import css from './MomentCard.scss';
+
+type MomentCardState = {};
+
+type MomentCardProp = {
+  item: {
+    name: string,
+    category?: [],
+    mood?: [],
+  },
+  date: string,
+  cardType: string,
+};
+
+export default class MomentCard extends React.Component<MomentCardProp, MomentCardState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.